modules/tinymce/src/core/main/ts/focus/EditorFocus.ts

```diff
@@ -119,6 +119,11 @@ const focusEditor = (editor: Editor) => {
     normalizeSelection(editor, rng);
   }
 
+  // This should be safe as the the focusin listener calls editor.focus(true) which means the focusEditor function is not called
+  // TODO: Should this also include the last blurredEditor?
```

Comment:
There's a couple of TODOs here. Are they still accurate? If so could we please make sure to either, fix, log a JIRA or remove them.

Reply:
Have removed the code and TODOs here